src/Tizen.Multimedia/AudioManager/AudioDevice.cs

```diff
@@ -104,9 +107,252 @@ public bool IsRunning
+                int[] formatsResult = new int[numberOfElements];
+
+                Marshal.Copy(formats, formatsResult, 0, (int)numberOfElements);
+                Interop.Libc.Free(formats);
+
+                foreach (int f in formatsResult)
+                {
+                    Log.Debug(Tag, $"supported sample format:{f}");
+                }
+
+                return formatsResult.Cast<AudioSampleFormat>();
+            }
+        }
+
+        /// <summary>
+        /// Sets the device's sample format.
+        /// </summary>
+        /// <value>The <see cref="AudioSampleFormat"/> of the device.</value>
+        /// <remarks>
+        /// This device should be <see cref="AudioDeviceType.UsbAudio"/> type and <see cref="AudioDeviceIoDirection.Output"/> direction.
+        /// <exception cref="ArgumentException">This device is not valid or is disconnected.</exception>
+        /// <exception cref="InvalidOperationException">This device is not <see cref="AudioDeviceType.UsbAudio"/> type or is not <see cref="AudioDeviceIoDirection.Output"/> direction.</exception>
+        /// </remarks>
+        /// <since_tizen> 5 </since_tizen>
+        public void SetSampleFormat(AudioSampleFormat format)
+        {
+            Interop.AudioDevice.SetSampleFormat(_id, format).
+                    ThrowIfError("Failed to set sample format of the device");
+        }
+
+        /// <summary>
+        /// Gets the device's sample format.
+        /// </summary>
+        /// <value>The <see cref="AudioSampleFormat"/> of the device.</value>
+        /// <remarks>
+        /// This device should be <see cref="AudioDeviceType.UsbAudio"/> type and <see cref="AudioDeviceIoDirection.Output"/> direction.
+        /// <exception cref="ArgumentException">This device is not valid or is disconnected.</exception>
+        /// <exception cref="InvalidOperationException">This device is not <see cref="AudioDeviceType.UsbAudio"/> type or is not <see cref="AudioDeviceIoDirection.Output"/> direction.</exception>
+        /// </remarks>
+        /// <since_tizen> 5 </since_tizen>
+        public AudioSampleFormat GetSampleFormat()
+        {
+            Interop.AudioDevice.GetSampleFormat(_id, out AudioSampleFormat format).
+                ThrowIfError("Failed to get sample format of the device");
+
+            return format;
+        }
+
+        private uint ConvertCoreRateValToUint(int value)
+        {
+            switch (value)
+            {
+                case 0: return 8000;
+                case 1: return 16000;
+                case 2: return 22050;
+                case 3: return 44100;
+                case 4: return 48000;
+                case 5: return 88200;
+                case 6: return 96000;
+                case 7: return 192000;
+                default:
+                    Log.Error(Tag, $"unknown value from core:{value}");
+                    return 0;
+            }
+        }
+
+        private uint ConvertRateToCoreValue(uint rate)
```

Comment:
It causes some performance advantage. Please refer below full deaciption of CA1822.

Members that do not access instance data or call instance methods can be marked as static (Shared in Visual Basic). After you mark the methods as static, the compiler will emit nonvirtual call sites to these members. Emitting nonvirtual call sites will prevent a check at runtime for each call that makes sure that the current object pointer is non-null. This can achieve a measurable performance gain for performance-sensitive code. In some cases, the failure to access the current object instance represents a correctness issue.

Reply:
Ok, I'll apply it as static considering the performance advantage.

Reply:
Please check below comment before you change this.

Reply:
Sure, I'm considering your comments including below.
Seems to think which is more useful for user.